docs/src/paradox/03-apis/api-admin/projects.md

```diff
@@ -0,0 +1,144 @@
+- `DELETE: /admin/projects/[identifier]` : update project status to false
+
+**Project Member Operations:**
+- `GET: /admin/projects/members/[identifier]` : returns all members part of a project identified through iri, shortname or shortcode
+
+**Project Admin Member Operations:**
+- `GET: /admin/projects/admin-members/[identifier]` : returns all admin members part of a project identified through iri, shortname or shortcode
+
+**Project Keyword Operations:**
+- `GET: /admin/projects/keywords` : returns all unique keywords for all projects as a list
+- `GET: /admin/projects/keywords/[identifier]` : returns all keywords for a single project
+
+**Project Restricted View Settings Operations:**
+- `GET: /admin/projects/[identifier]/RestrictedViewSettings` : returns the project's restricted view settings
+
+
+## Project Operations
+
+### Create a new project:
+
+  - Required permission: SystemAdmin
+  - Required information: shortname (unique; used for named graphs),
+    status, selfjoin
+  - Optional information: longname, description, keywords, logo
+  - Returns information about the newly created project
+  - Remark: There are two distinct use cases / payload combination:
+    (1) change ontology and data graph: ontologygraph, datagraph,
+    (2) basic project information: shortname, longname, description,
+    keywords, logo, institution, status, selfjoin
+  - TypeScript Docs: projectFormats - CreateProjectApiRequestV1
+  - POST: `/admin/projects/`
+  - BODY:
+
+```JSON
+{
+  "shortname": "newproject",
+  "longname": "project longname",
+  "description": "project description",
+  "keywords": "keywords",
+  "logo": "/fu/bar/baz.jpg",
+  "status": true,
+  "selfjoin": false
+}
+```
+
+### Update project information:
+
+  - Required permission: SystemAdmin / ProjectAdmin
+  - Changeable information: shortname, longname, description,
+    keywords, logo, status, selfjoin
+  - TypeScript Docs: projectFormats - ChangeProjectApiRequestV1
+  - PUT: `/admin/projects/<projectIri>`
+  - BODY:
+
+```JSON
+{
+  "shortname": "newproject",
+  "longname": "project longname",
+  "description": "project description",
+  "keywords": "keywords",
+  "logo": "/fu/bar/baz.jpg",
+  "status": true,
+  "selfjoin": false
+}
+```
+
+### Delete project (update project status):
+
+  - Required permission: SystemAdmin / ProjectAdmin
+  - Remark: The same as updating a project and changing `status` to
+    `false`. To un-delete, set `status` to `true`.
+  - DELETE: `/admin/projects/<projectIri>`
+  - BODY: empty
+
+
+## Project Member Operations
+
+### Get project members:
+
+  - Required permission: SystemAdmin / ProjectAdmin
+  - Required information: project IRI
+  - GET: `/admin/projects/members/<projectIri>`
+
+
+### Restricted View Settings Operations
+
+Operates on the following properties:
+ - `knora-base:projectRestrictedViewSize` - takes the IIIF size value
+ - `knora-base:projectRestrictedViewWatermark` - takes the path to the watermark image. **Currently not used.**
```

Comment:
Would it be possible to specify that I want restricted view to have:

* A size without a watermark
* A watermark without a size
* Both a size and a watermark

Reply:
Currently, Sipi allows only one or the other. Also,  only size is implemented:

https://github.com/dhlab-basel/Knora/blob/663b1b1bdc3e244b694c621ecfc61a9eab93a096/sipi/config/sipi.init-knora.lua#L139-L141

As soon as Sipi allows both, then whatever combination is specified in the project, will also be returned by Sipi:

- only size
- only watermark
- both size and watermark

Reply:
OK!